azure_pipelines/changes.txt

```diff
@@ -2,4 +2,5 @@
 Added git config default user. Adding azure pipelines to apps allowed to push
```

Comment:
This pull request does not update CHANGELOG.md.

Please consider if this change would be noticeable to a partner or user and either update CHANGELOG.md or resolve this conversation.
<!-- changelog-comment -->